command_line/symmetry.py

```diff
@@ -55,6 +65,21 @@
   .type = float
   .help = "Use only reflections with a partiality above this threshold."
 
+mode = *laue_plus_absences laue_only absences_only
+  .type = choice
```

Comment:
I wonder if this parameter could be improved slightly, as the current options seem a bit verbose. Two possibilities:
1)
```
mode = *laue *absences
  .type = choice(multi=True)
```
This would translate to the following command lines:
```
$ dials.symmetry DataFiles/AUTOMATIC_DEFAULT_scaled.{expt,refl} mode=absences
$ dials.symmetry DataFiles/AUTOMATIC_DEFAULT_scaled.{expt,refl} mode=laue
$ dials.symmetry DataFiles/AUTOMATIC_DEFAULT_scaled.{expt,refl} mode="*absences *laue"
```
The last line here is a bit cumbersome as it necessitates both quoting and selecting both modes with an asterisk (although this is the default anyway).

2)
```
mode = *full laue absences
  .type = choice
```
Which translates to the following command lines:
```
$ dials.symmetry DataFiles/AUTOMATIC_DEFAULT_scaled.{expt,refl} mode=absences
$ dials.symmetry DataFiles/AUTOMATIC_DEFAULT_scaled.{expt,refl} mode=laue
$ dials.symmetry DataFiles/AUTOMATIC_DEFAULT_scaled.{expt,refl} mode=full
```

Reply:
I like option 2 :)

Reply:
Another alternative would be:
```
laue_group = auto
  .type = space_group
systematic_absences = True
  .type = bool
```
Command lines:
```
# Use input laue_group, do systematic absences check
$ dials.symmetry scaled.{expt,refl} laue_group=None systematic_absences=True

# Use laue_group from command line, do systematic absences check
$ dials.symmetry scaled.{expt,refl} laue_group=P222 systematic_absences=True

# Switch off systematic absences check
$ dials.symmetry scaled.{expt,refl} systematic_absences=False

# Laue group + systematic absences check (default)
$ dials.symmetry scaled.{expt,refl} laue_group=auto systematic_absences=True
```
This option would be more closely aligned with the [LAUEGROUP](http://www.ccp4.ac.uk/html/pointless.html#lauegroup) and [SYSTEMATICABSENCE](http://www.ccp4.ac.uk/html/pointless.html#sysabs) keywords in pointless.